cypress/e2e/tests/custom-metrics/issues_tracker.test.ts

```diff
@@ -0,0 +1,125 @@
+import { JiraCredentials } from "../../models/administration/credentials/JiraCredentials";
+import { Metrics } from "../../models/migration/custom-metrics/custom-metrics";
+
+const now = new Date();
+now.setDate(now.getDate() + 1);
+
+const end = new Date(now.getTime());
+end.setFullYear(end.getFullYear() + 1);
+
+let applications: Assessment[];
+let migrationWave: MigrationWave;
+let jiraInstance: Jira;
+let jiraCredentials: JiraCredentials;
+const metrics = new Metrics();
+const metricName = "konveyor_issues_exported_total";
+let counter: number;
+
+// Automates Polarion TC 337
+describe(["@tier2"], "Custom Metrics - Count the total number of issues exported", function () {
+    before("Create test data", function () {
+        login();
+        applications = createMultipleApplications(2);
+
+        migrationWave = new MigrationWave(
+            data.getRandomWord(8),
+            now,
+            end,
+            null,
+            null,
+            applications
+        );
+        migrationWave.create();
+
+        jiraCredentials = new JiraCredentials(
+            data.getRandomCredentialsData(CredentialType.jiraBasic, null, true)
+        );
+        jiraCredentials.create();
+
+        jiraInstance = new Jira({
+            name: data.getRandomWord(5),
+            url: Cypress.env("jira_atlassian_cloud_url"),
+            credential: jiraCredentials,
+            type: JiraType.cloud,
+        });
+        jiraInstance.create();
+
+        metrics.getValue(metricName).then((counterValue) => {
+            counter = counterValue;
+        });
+    });
+
+    it("Export issues to Jira and validate the metrics counter", function () {
+        let projectName = "";
+
+        jiraInstance
+            .getProject()
+            .then((project) => {
+                projectName = project.name;
+
+                return jiraInstance.getIssueType("Task");
+            })
+            .then((task) => {
+                migrationWave.exportToIssueManager(
+                    JiraType.cloud,
+                    jiraInstance.name,
+                    projectName,
+                    task.untranslatedName
+                );
+            })
+            .then((_) => {
+                cy.wait(35 * SEC); // Enough time to create both tasks and for them to be available in the Jira API
+                return jiraInstance.getIssues(projectName);
+            })
+            .then((issues: JiraIssue[]) => {
+                expect(
+                    migrationWave.applications.every((app) =>
+                        issues.find((issue) => issue.fields.summary.includes(app.name))
+                    )
+                ).to.eq(true);
+                counter++;
+                counter++;
+
+                // Validate the issues exported count change
+                metrics.validateMetric(metricName, counter);
+
+                // Delete the migration wave
+                migrationWave.delete();
```

Comment:
**suggestion:** The line  (`migrationWave.delete()`) is clear enough so there is no need to add this kind of comment.

We have to avoid adding noise to the code. More info: https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29#comments-rules